src/style.css

```
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
```

```suggestion
.stars{
```
Можна просто створити клас stars для цього, і використовувати його в комбінації з модифікатором
```
<div class="stars stars--5">
```